harness/determined/exec/prep_container.py

```diff
@@ -233,8 +233,20 @@ def do_proxy(sess: Session, allocation_id: str) -> None:
     parser.add_argument("--resources", action="store_true")
     parser.add_argument("--rendezvous", action="store_true")
     parser.add_argument("--proxy", action="store_true")
+    parser.add_argument(
+        "--log-level",
+        default=os.getenv("DET_LOG_LEVEL", "INFO"),
+        choices=["DEBUG", "INFO", "WARNING", "ERROR"],
+        help="Set the logging level",
+    )
```

Comment:
instead of making this configurable as a cli argument (the user cannot control which cli arguments get passed here), can you make it either INFO or DEBUG, based on the value of `info._debug`, which comes from the experiment config?